views/login.html

```diff
@@ -5,10 +5,10 @@
 {% endblock %}
 
 {% block header %}
-<header class="main-header">
+<header class="header _header">
```

Comment:
@poofichu If `_header` is for positioning, shouldn't `header` be for visual styling?